xdsl/interactive/app.py

```diff
@@ -554,6 +565,8 @@ def check_load_file(file_path: str) -> None:
                     with open(file_path) as file:
                         file_contents = file.read()
                         self.input_text_area.load_text(file_contents)
+                    self.pre_loaded_file_path = file_path
```

Comment:
```suggestion
                    self.current_file_path = file_path
```
It feels like the name is a bit misleading, if we update it when the user interacts with the app, then `pre_loaded` no longer applies.